config/routes.rb

```diff
@@ -33,12 +33,11 @@
       resources :recruitment_cycles, param: :year, constraints: { year: /#{Settings.current_recruitment_cycle_year}|#{Settings.current_recruitment_cycle_year + 1}/ }, path: "", only: [] do
         get "/about", on: :member, to: "providers#about"
         put "/about", on: :member, to: "providers#update"
-        get "/contact", on: :member, to: "providers#contact"
-        put "/contact", on: :member, to: "providers#update"
         get "/details", on: :member, to: "providers#details"
 
         scope module: :providers do
           resource :visas, only: %i[edit update], path: "/visas"
+          resource :contact, only: %i[edit update], path: "/contact"
```

Comment:
Worth bearing in mind - this alters the path structure from Old Publish slightly (GET `/contact` in Old Publish vs. GET `/contact/edit` here). Not a huge deal but potentially makes writing controller redirects in Old Publish a bit more fiddly.

Reply:
How fiddly? I think we should try to keep the new routes more RESTful. There's a bit of an anti-pattern we'd be porting over with the old routes and if there's an opportunity to improve them without too much cost I'd rather take advantage of it now

Reply:
users may have bookmarked these

Reply:
Add a simple redirect in this app from the old path to the new more restful URL?

Reply:
I've just kept this backwards compatible for now https://github.com/DFE-Digital/teacher-training-api/pull/2358/commits/0e48b9b00e5f68204193f774f615192d56ed5f93, we can improve the routes after the migration